WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js

```diff
@@ -2678,7 +2680,7 @@ function(self, attempt) {
 ZmMailMsgView.prototype.setScrollWithIframe =
 function(val) {
 	
-	if (!this._usingIframe) { return; }
+	if (!this._usingIframe && !val) { return; }
```

Comment:
do we need this change?

Reply:
yes, because in message view we have iframe every-time but still this flag is not working. so added additional condition.